client/my-sites/domains/domain-management/components/primary-domain-selector/style.scss

```
@@ -0,0 +1,17 @@
+.domains-set-primary-address {
+	margin-bottom: 2em;
+	box-shadow: 0 0 0 1px var(--color-border-subtle);
+	padding: 24px;
+	&__title {
+		margin-bottom: 0;
+		/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+		line-height: 1.5em;
+		font-weight: 600;
+		color: var(--studio-gray-80);
+	}
+	&__subtitle {
+		font-size: $font-body-small;
+		margin-bottom: 1em;
+		color: var(--studio-gray-50);
```

Nit: this is different on some colour schemes, and probably the more appropriate variable.

```suggestion
		color: var(--color-text);
	}
	&__subtitle {
		font-size: $font-body-small;
		margin-bottom: 1em;
		color: var(--color-text-subtle);
```

Although to be honest, I'm not even sure that the colour needs to be set for the title. 

This is also less relevant if it's changed to `FormFieldset`.